docs/text/rolling.rst

```diff
@@ -21,6 +21,11 @@ the last 10 steps.
 A rolling mechanism creates such time series for every time step by creating sub time series of the sensor data of the
 last 10 time steps.
 
+<!---
+The above scenario cannot be solved by tsfresh, can it?
```

Comment:
Sorry, I missed answering this :-)
Yes, if it would be possible to include this (maybe as a parameter) to the already implemented rolling function, this would be very neat, If you have time, feel free to do so. If not, I can work on this.